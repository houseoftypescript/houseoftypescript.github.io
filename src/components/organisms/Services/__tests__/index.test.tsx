import { render } from '@testing-library/react';
import { Services } from '../index';

describe('Services', () => {
  it('renders correctly', () => {
    const button = render(
      <Services title={''} description={''} services={[]} />
    );
    expect(button.container).toMatchSnapshot();
  });
});
