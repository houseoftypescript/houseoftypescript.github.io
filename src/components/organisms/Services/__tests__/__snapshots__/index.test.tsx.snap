// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Services renders correctly 1`] = `
<div>
  <section
    class="bg-[#FFF5ED]/50 py-8 md:py-16"
    id="services"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="mb-8 md:mb-16"
      >
        <div
          class="text-center max-w-lg mx-auto"
        >
          <h1
            class="text-xl sm:text-3xl mb-4 font-b162 font-bold"
          />
          <h2
            class="text-sm sm:text-lg"
          />
        </div>
      </div>
      <div
        class="grid grid-cols-1 sm:grid-cols-3 gap-8"
      />
    </div>
  </section>
</div>
`;
